src/tokenizer.ts

```diff
@@ -1697,6 +1697,18 @@ export class Tokenizer extends DiagnosticEmitter {
     return this.readHexadecimalEscape(4, startIfTaggedTemplate);
   }
 
+  hasNoMoreExpressionInLine(): bool {
+    let pos = this.pos;
+    const end = this.end;
+    do {
+      const c = this.source.text.charCodeAt(pos);
```

Comment:
Looks like `pos` can be `== end` here, so the access can fail. A `while` instead would probably work.

Reply:
fixed